api_functional_v2_test.go

```diff
@@ -31,6 +31,31 @@ import (
 	"github.com/minio/minio-go/pkg/policy"
 )
 
+const letterBytes = "abcdefghijklmnopqrstuvwxyz01234569"
```

Comment:
only few because it is generally just to validate that if v2 is working fine with most APIs if not all. We can add you can confirm this with @abperiasamy . IMO we should include v2 if we support it on server side.